tests/store/test_gcs_artifact_repo.py

```diff
@@ -33,34 +33,41 @@ def test_list_artifacts_empty(gcs_mock):
     assert repo.list_artifacts() == []
 
 
-def test_list_artifacts(gcs_mock):
-    dest_path = "/some/path/"
+def test_list_artifacts_with_subdir(gcs_mock):
+    dest_path = "/experiment_id/run_id/"
     repo = GCSArtifactRepository("gs://test_bucket" + dest_path, gcs_mock)
 
+    # mocked bucket/blob structure
+    # gs://test_bucket/experiment_id/run_id/model
+    #  | model.pb
+    #  | variables
+
     # mocking a single blob returned by bucket.list_blobs iterator
     # https://googlecloudplatform.github.io/google-cloud-python/latest/storage/buckets.html#google.cloud.storage.bucket.Bucket.list_blobs
-    dir_mock = mock.Mock()
-    dir_name = "0_subpath"
-    dir_mock.configure_mock(prefixes=(dest_path + dir_name + "/",))
+    dir_name = "model"
 
     obj_mock = mock.Mock()
-    file_name = '1_file'
+    file_name = dir_name + "/" + 'model.pb'
     obj_mock.configure_mock(name=dest_path + file_name, size=1)
 
+    subdir_mock = mock.Mock()
+    subdir_name = dir_name + "/" + 'variables'
+    subdir_mock.configure_mock(prefixes=(dest_path + subdir_name + "/",))
```

Comment:
Does the trailing slash in the `prefixes` attribute match what GCS returns?

Reply:
yes this is what is being returned